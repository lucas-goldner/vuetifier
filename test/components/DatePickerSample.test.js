import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import target from '@/components/DatePickerSample'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)

describe('index', () => {
  it('should be vue instance', () => {
    const vuetify = new Vuetify()
    const wrapper = shallowMount(target, {
      localVue,
      vuetify,
      propsData: {
        value: new Date(2019, 1, 1, 9).toISOString().substr(0, 10),
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
