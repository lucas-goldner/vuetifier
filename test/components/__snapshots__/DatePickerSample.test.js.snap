// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should be vue instance 1`] = `
"<v-layout-stub tag=\\"div\\" wrap=\\"\\" align-center=\\"\\" justify-center=\\"\\"><v-flex-stub tag=\\"div\\" text-center=\\"\\" xs12=\\"\\"><div class=\\"title-text\\">
      Datepicker
    </div> <link-buttons-stub document=\\"https://vuetifyjs.com/en/components/date-pickers\\" source=\\"https://github.com/reireias/vuetifier/blob/master/components/DatePickerSample.vue\\"></link-buttons-stub></v-flex-stub> <v-flex-stub tag=\\"div\\" text-center=\\"\\" xs12=\\"\\"><v-date-picker-stub headercolor=\\"primary\\" width=\\"290\\" eventcolor=\\"warning\\" firstdayofweek=\\"0\\" nexticon=\\"$vuetify.icons.next\\" previcon=\\"$vuetify.icons.prev\\" showcurrent=\\"true\\" selecteditemstext=\\"$vuetify.datePicker.itemsSelected\\" type=\\"date\\" value=\\"2019-02-01\\"></v-date-picker-stub></v-flex-stub></v-layout-stub>"
`;
